ai/src/main/java/com/alibaba/nacos/ai/service/McpServerOperationService.java

```diff
@@ -544,4 +564,201 @@ private String resolveMcpServerId(String namespaceId, String serverName, String
         
         return null;
     }
+    
+    /**
+     * Validate servers for import.
+     *
+     * @param namespaceId namespace ID
+     * @param request import request
+     * @return validation result
+     * @throws NacosException if validation fails
+     */
+    public McpServerImportValidationResult validateImport(String namespaceId, McpServerImportRequest request) 
```

Comment:
I suggest you abstract import logic into a `McpServerImportService` not in `OperationService`

`OperationService` should do atomic operation for mcp server.